spec/features/assignments/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'viewing the calendar' do
+  let(:roster) { create :roster }
+  let(:user) { create :user, rosters: [roster] }
+  before :each do
+    Timecop.freeze Date.new(2017, 8, 14)
+    set_current_user(user)
+  end
+
+  it 'highlights today' do
+    visit roster_assignments_url(roster)
+    expect(page).to have_selector('td.cal-cell.current-day', text: '14')
+  end
+  context 'assignment belongs to user' do
+    it 'appears highlighted yellow for your assignment' do
+      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
```

Comment:
The `2.days.since` syntax is actually just ActiveSupport. But we can use it and always know how it will behave thanks to Timecop :-)